src/third_party/starboard/wpe/amlogic/armv7ahf/gyp_configuration.gypi

```diff
@@ -0,0 +1,193 @@
+# Copyright 2016 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{
+  'includes': [
+    '../../shared/libraries.gypi',
+    'architecture.gypi',
+  ],
+  'variables': {
+    'target_arch': 'arm',
+    'target_os': 'linux',
+
+    'sysroot%': '/',
+    'gl_type': 'system_gles2',
+
+    # This is to create cobalt shared library
+    'final_executable_type': '<!(echo $COBALT_EXECUTABLE_TYPE)',
+
+    'platform_libraries': [
+      '<@(common_libs)',
+    ],
+
+    'linker_flags_host': [ '-pthread' ],
+
+    # Define platform specific compiler and linker flags.
+    # Refer to base.gypi for a list of all available variables.
+    'compiler_flags_host': [
+      '-O2',
+    ],
+    'compiler_flags': [
+      # We'll pretend not to be Linux, but Starboard instead.
+      '-U__linux__',
+
+      # Force char to be signed.
+      '-fsigned-char',
+      # Disable strict aliasing.
+      '-fno-strict-aliasing',
+
+      # To support large files
+      '-D_FILE_OFFSET_BITS=64',
+
+      # Suppress some warnings that will be hard to fix.
+      '-Wno-unused-local-typedefs',
+      '-Wno-unused-result',
+      '-Wno-unused-function',
+      '-Wno-deprecated-declarations',
+      '-Wno-missing-field-initializers',
+      '-Wno-extra',
+      '-Wno-comment',  # Talk to my lawyer.
+      '-Wno-narrowing',
+      '-Wno-unknown-pragmas',
+      '-Wno-type-limits',  # TODO: We should actually look into these.
+      # It's OK not to use some input parameters. Note that the order
+      # matters: Wall implies Wunused-parameter and Wno-unused-parameter
+      # has no effect if specified before Wall.
+      '-Wno-unused-parameter',
+
+      # Specify the sysroot with all your include dependencies.
+      '--sysroot=<(sysroot)',
+
+      # GStreamer includes
+      '-I<(sysroot)/usr/include/gstreamer-1.0',
+      '-I<(sysroot)/usr/include/glib-2.0',
+      '-I<(sysroot)/usr/lib/glib-2.0/include/',
+      '-I<(sysroot)/usr/include/WPEFramework/compositor/',
```

Comment:
Here these includes are not required.
Recently Cobalt is modified to use pkg-config to read libraries and includes to be used.